composer.json

```diff
@@ -22,49 +22,52 @@
-        "doctrine/persistence": "^1.3.3",
-        "sonata-project/admin-bundle": "^3.35",
-        "sonata-project/block-bundle": "^3.18",
-        "sonata-project/cache": "^1.0.2 || ^2.0",
-        "sonata-project/cache-bundle": "^2.4",
-        "sonata-project/core-bundle": "^3.14",
-        "sonata-project/datagrid-bundle": "^2.3",
-        "sonata-project/doctrine-extensions": "^1.1.3",
-        "sonata-project/doctrine-orm-admin-bundle": "^3.5",
+        "doctrine/persistence": "^1.3.4",
+        "sonata-project/admin-bundle": "^3.70.1",
+        "sonata-project/block-bundle": "^3.20",
+        "sonata-project/cache": "^1.0.3 || ^2.0",
+        "sonata-project/cache-bundle": "^2.4.1",
+        "sonata-project/datagrid-bundle": "^2.5",
+        "sonata-project/doctrine-extensions": "^1.5",
+        "sonata-project/doctrine-orm-admin-bundle": "^3.19",
         "sonata-project/easy-extends-bundle": "^2.5",
-        "sonata-project/notification-bundle": "^3.3",
-        "sonata-project/seo-bundle": "^2.5",
-        "symfony-cmf/routing-bundle": "^1.1 || ^2.0",
-        "symfony/config": "^4.3",
-        "symfony/console": "^4.3",
-        "symfony/debug": "^4.3",
-        "symfony/dependency-injection": "^4.3",
-        "symfony/form": "^4.3",
-        "symfony/http-foundation": "^4.3.8",
-        "symfony/http-kernel": "^4.3",
-        "symfony/options-resolver": "^4.3",
-        "symfony/process": "^4.3",
-        "symfony/routing": "^4.3",
-        "symfony/security-core": "^4.3",
-        "symfony/security-http": "^4.3",
-        "symfony/templating": "^4.3",
-        "symfony/validator": "^4.3",
+        "sonata-project/form-extensions": "^0.1.1 || ^1.4",
+        "sonata-project/notification-bundle": "^3.8",
+        "sonata-project/seo-bundle": "^2.11",
+        "sonata-project/twig-extensions": "^0.1.1 || ^1.3",
+        "symfony-cmf/routing-bundle": "^2.1",
+        "symfony/config": "^4.4",
+        "symfony/console": "^4.4",
+        "symfony/debug": "^4.4",
+        "symfony/dependency-injection": "^4.4.3",
+        "symfony/form": "^4.4",
+        "symfony/http-foundation": "^4.4",
+        "symfony/http-kernel": "^4.4",
+        "symfony/options-resolver": "^4.4",
+        "symfony/process": "^4.4",
+        "symfony/routing": "^4.4",
+        "symfony/security-core": "^4.4",
+        "symfony/security-http": "^4.4",
+        "symfony/templating": "^4.4",
+        "symfony/validator": "^4.4",
         "twig/string-extra": "^3.0",
         "twig/twig": "^2.12.1"
     },
     "conflict": {
         "friendsofsymfony/rest-bundle": "<2.1 || >=3.0",
-        "jms/serializer": "<0.13"
+        "jms/serializer": "<0.13",
+        "sonata-project/core-bundle": "<3.20"
     },
     "require-dev": {
         "friendsofsymfony/rest-bundle": "^2.1",
-        "jms/serializer-bundle": "^1.0 || ^2.0",
+        "jms/serializer-bundle": "^2.0 || ^3.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
```

Comment:
Bumping this one solves many issues. I tried `composer update --prefer-lowest` on this branch, and after updating it, I'm left with one issue.

Reply:
The other issue can be fixed by updating `doctrine/annotations`

Reply:
I've got some time, I'll do it for you :)

Reply:
@greg0ire Thanks. I do it in other bundles but here i miss it.